packages/cli/README-source.md

```diff
@@ -1415,15 +1424,18 @@ Example: `throw new z.errors.HaltedError('Your reason.');`
 
 For apps that require manual refresh of authorization on a regular basis, Zapier
 provides a mechanism to notify users of expired credentials. With the
-`ExpiredAuthError`, the current operation is interrupted, the Zap is turned off
-(to prevent more calls with expired credentials), and a predefined email is sent
-out informing the user to refresh the credentials.
+`ExpiredAuthError`, the current operation is interrupted and a predefined email
+is sent out asking the user to refresh the credentials. While the auth is
+disconnected, Zap runs will not be executed, to prevent more calls with expired
```

Comment:
I think it's still necessary to mention the Zap is turned off when the auth is disconnected:

```suggestion
disconnected, the Zap will be turned off and Zap runs will not be executed, to prevent more calls with expired
```

Reply:
@eliangcs The reason I changed that is that it's not correct. The Zap is not turned off. 😄 Instead, it remains on and potential runs are held, as described.

Reply:
@armchairlinguist glad you tested it! No need to change then 👍